staff/henry-novoa/cv/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+    <h1>Curriculum Vitae</h1>
+
+    <section id="personal data">
+      <h2>Personal Data</h2>
+        <p>Name: Henry Novoa</p>
+    <address>
+        <P>Date of birth: 8th of June, 1990</p>
+        <P>Place of birth: New York, USA</P>    
+        <p>Current location: Barcelona</p>
+    </address>
+    
+    </section>
+
+    <section id="work experience">
```

Comment:
if you want to add an ID longer than 2 words you should try to write them like: workExperience

Reply:
do not use id's for no reason. id's should be avoided as much as possible, and semantics only should be enough.

Reply:
ids should be avoided, they are not part of semantics